charts/frameSkipChart.js

```diff
@@ -0,0 +1,42 @@
+import Chart from './baseChart';
+
+module.exports = class FrameSkipResultsChart extends Chart {
```

Comment:
Instead of using prototypal inheritance, would it be possible for these charts to pass in props to BaseChart? I feel it might be more declarative and goes with the idea of using components as building blocks. I haven't thought about implementation details of what to pass in so I'm uncertain how feasible it would be.

Reply:
i don't see how it would be more declarative or be more building-blocky then it is now, is there an example you could show as to how it would be clearer if it was done using the method you're talking about?

Reply:
```
return (
  <BaseChart
      title = 'title'
      series = { [ series ] }
    ... />
```

Reply:
ah! i see.  yeah that might be better.  i'll play with that a bit.

Reply:
turns out this is a bit tough because then i can't leverage the helper methods i have in `BaseChart` for rendering loading/error/data and fetching the results.